ResearchKit/Common/ORKImageCaptureView.m

```diff
@@ -119,6 +115,7 @@ - (void)orientationDidChange {
     }
     [_previewView setVideoOrientation:orientation];
     [self.delegate videoOrientationDidChange:orientation];
+    [self setNeedsUpdateConstraints];
```

Comment:
The rest of the changes are nice enough but not needed for release. I'd like a new PR direct to the convergence branch which adds this line only.